policy-controller/src/lib.rs

```diff
@@ -113,18 +119,23 @@ impl DiscoverOutboundPolicy<OutboundDiscoverTarget> for OutboundDiscover {
     async fn watch_outbound_policy(
         &self,
         OutboundDiscoverTarget {
-            service_name,
-            service_namespace,
-            service_port,
+            kind,
+            name,
+            namespace,
+            port,
             source_namespace,
         }: OutboundDiscoverTarget,
     ) -> Result<Option<OutboundPolicyStream>> {
-        match self.0.write().outbound_policy_rx(
-            service_name,
-            service_namespace,
-            service_port,
-            source_namespace,
-        ) {
+        if let TargetKind::UnmeshedNetwork = kind {
+            tracing::debug!("UnmeshedNetwork network is not supported yet");
```

Comment:
nit: this message should probably match the `debug!` event above in `get_outbound_policy`, for consistency.